commands.json

```diff
@@ -3547,6 +3547,52 @@
     "since": "2.0.0",
     "group": "sorted_set"
   },
+  "ZDIFF": {
+    "summary": "Subtract multiple sorted sets",
+    "complexity": "O(N)+O(M*log(M)) worst case with N is the total number of elements in all given sets and M being the number of elements in the resulting sorted set.",
```

Comment:
@itamarhaber i noticed that [SDIFF](https://redis.io/commands/sdiff) documentation just mentions:
```
Time complexity: O(N) where N is the total number of elements in all given sets.
```
although the code says:
```c
     * Algorithm 1 is O(N*M) where N is the size of the element first set
     * and M the total number of sets.
     *
     * Algorithm 2 is O(N) where N is the total number of elements in all
     * the sets.
```
do you think it's because the documentation wasn't updated when the algorithm was improved? or is it trying to state the worse case in a more simplistic way?